packages/common/src/services/audius-backend/solana.ts

```diff
@@ -282,3 +312,94 @@ export const purchaseContent = async (
     await audiusBackendInstance.getAudiusLibs()
   ).solanaWeb3Manager!.purchaseContent(config)
 }
+
+export const findAssociatedTokenAddress = async (
+  audiusBackendInstance: AudiusBackend,
+  { solanaAddress, mint }: { solanaAddress: string; mint: MintName }
+) => {
+  return (
+    await audiusBackendInstance.getAudiusLibsTyped()
+  ).solanaWeb3Manager!.findAssociatedTokenAddress(solanaAddress, mint)
+}
+
+export const createTransferToUserBankTransaction = async (
```

Comment:
It seems like we had the convention of doing transaction creation and processing in `SolanaWeb3Manager` / libs level code. I'm curious why we're doing it up at this layer instead?

Reply:
This isn't shared outside our clients, so there's not a real reason imo to put it in libs